src/dsp_tools/utils/xmlupload/upload_stashed_resptr_props.py

```diff
@@ -35,177 +33,97 @@ def upload_stashed_resptr_props(
 
     print("Upload the stashed resptrs...")
     logger.info("Upload the stashed resptrs...")
-    nonapplied_resptr_props = stashed_resptr_props.copy()
-    for resource, prop_2_resptrs in stashed_resptr_props.items():
-        if resource.id not in id2iri_mapping:
+    not_uploaded: list[LinkValueStashItem] = []
+    for res_id, stash_items in stashed_resptr_props.res_2_stash_items.items():
+        if res_id not in id2iri_mapping:
             # resource could not be uploaded to DSP, so the stash cannot be uploaded either
             # no action necessary: this resource will remain in nonapplied_resptr_props,
             # which will be handled by the caller
             continue
-        res_iri = id2iri_mapping[resource.id]
+        res_iri = id2iri_mapping[res_id]
         try:
             existing_resource = try_network_action(con.get, route=f"/v2/resources/{quote_plus(res_iri)}")
         except BaseError as err:
-            # Apart from that, no action is necessary:
-            # this resource will remain in nonapplied_resptr_props, which will be handled by the caller
-            _log_if_unable_to_retrieve_resource(err=err, resource=resource)
+            _log_if_unable_to_retrieve_resource(err, res_id)
             continue
-        print(f'  Upload resptrs of resource "{resource.id}"...')
-        logger.info(f'  Upload resptrs of resource "{resource.id}"...')
-        for link_prop, resptrs in prop_2_resptrs.items():
-            nonapplied_resptr_props = _upload_all_resptr_props_of_single_resource(
-                resource_in_triplestore=existing_resource,
-                stashed_resource=resource,
-                link_prop=link_prop,
-                res_iri=res_iri,
-                resptrs=resptrs,
-                id2iri_mapping=id2iri_mapping,
-                con=con,
-                nonapplied_resptr_props=nonapplied_resptr_props,
-                verbose=verbose,
-            )
-
-    # make a purged version of nonapplied_resptr_props, without empty entries
-    nonapplied_resptr_props = purge_stashed_resptr_props(
-        stashed_resptr_props=nonapplied_resptr_props,
-        id2iri_mapping=id2iri_mapping,
-    )
-    return nonapplied_resptr_props
-
-
-def _upload_all_resptr_props_of_single_resource(
-    resource_in_triplestore: dict[str, Any],
-    stashed_resource: XMLResource,
-    link_prop: XMLProperty,
+        if verbose:
+            print(f'  Upload resptrs of resource "{res_id}"...')
```

Comment:
Do you want to take this out the print statement in a future PR or could you do it now? At the moment, we do not get prints for Standoff and the console output is different for the two stashes.

Reply:
I tried to harmonize it, I think in both cases now, we get prints, if verbose=True... but I may be mistaken

Reply:
I haven't tried it out, but it'll be a quick fix in case it isn't.